paper.md

```diff
@@ -0,0 +1,127 @@
+is comprised of C/C++ developers, CMinx is designed to operate similar to
+Doxygen [@doxygen], the *de facto* C/C++ API documentation tool.
+Specifically, developers annotate their CMake source with "documentation"
+comments, which are traditional CMake block comments starting with an extra
+"`[`" character. The documentation comments, written in reST [@rest],
+describe to the reader how the functions, parameters, and variables should be
+used. Running CMinx on the annotated source code generates reST files containing
+the API documentation. The reST files can then be converted into static
+websites with tools such as Sphinx [@sphinx] or
+easily converted to another format via Pandoc [@pandoc].
+
+Unlike other solutions for documenting CMake modules, CMinx knows the CMake
+language's grammar. This enables CMinx to automatically extract function/macro
+signatures, even when functions are not documented. CMinx also integrates
+seamlessly into existing CMake build systems. CMinx's output is highly
+customizable and easily controlled via a YAML [@yaml] configuration file.
+CMinx has already proved to be an invaluable
+productivity tool in the authors' other projects, and, given that other
+scientific software projects also rely heavily on source code written in CMake,
+we anticipate CMinx will prove invaluable to many additional projects as well.
+
+Concurrent with the submission of this manuscript, we have also released the
+first production version of CMinx, version 1.0.0. CMinx can be obtained
+from the Python Packaging Index via "`pip install CMinx`". Alternatively, CMinx
+can be used as a CMake module via CMake's "`FetchContent`" command.
+Despite only just releasing 1.0.0, the CMinx GitHub organization
+has already started to see attention and interest from developers not
+affiliated with the authors. So while CMinx might not be considered research
+software, we anticipate CMinx to be a useful productivity tool for the large
+swath of research software that uses CMake as their build system.
+
+# Statement of need
+
+The process of building a software package written in a compiled language
+(*e.g.*, C or C++) has always been complicated. Over the years, various build
+system solutions have evolved to ease the process. Historically, there
+has been a propensity to treat each build system as a one-off use case. This is
+understandable since build systems have tended to be relatively small
+and tightly coupled to the structure and purpose of the package.
+With build system complexity at an all-time high[@snir2014,@xSDK], there is an
+increasing need to treat the underlying build system infrastructure as code.
+This means that the build system should be modularized, and those modules
+should be documented, tested, and reusable. With the popularity of C/C++ for
+high-performance computing, "build system" is increasingly becoming synonymous
+with CMake. Therefore with the advent of the exascale era, there is a desperate
```

Comment:
```suggestion
with CMake: there is a desperate
```
I think the need is there whether you're application targets the exascale or not.